Rework procedure to run Salt (#2458)
theforeman/foreman-documentation: guides/common/modules/proc_viewing-salt-reports.adoc

[id="Viewing_Salt_Reports_{context}"]
= Viewing Salt Reports

You can view uploaded Salt reports from Salt Minions in {Project}.

.Procedure

> * To view all Salt reports, in the {ProjectWebUI}, navigate to *Monitor* > *Config Management* and select a host.

Reviewer: Isn't this also for one host?

Author: True, reworded.

> * To view Salt reports associated with a host, in the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and select a host.
> You can view the Salt reports on the *Reports* tab.

Reviewer 2: ```suggestion
* To view Salt reports associated with a host, in the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*, select a host, and click the *Reports* tab.
```

Author: Applied.
